bot/src/main.rs

```diff
@@ -1455,7 +1495,11 @@ fn classify(
 fn main() -> BoxResult<()> {
     solana_logger::setup_with_default("solana=info");
 
-    let (config, rpc_client, mut stake_pool) = get_config()?;
+    let (config, rpc_client, stake_pool) = get_config()?;
```

Comment:
Rather than adding `commit` to the `Config` structure, we could just have `get_config` return the `command` as another return argument.

It would be nice to command to be an enum instead of a string, for better static typing, such as:
```rust
#[derive(PartialEq, Clone, Copy)]
enum Command {
  Export,
  Stake,
}
```
then we check for `if command == Command::Export {` here